eu.ad5001.LogarithmPlotter.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "eu.ad5001.LogarithmPlotter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.DBus.Proprieties",
+        "--talk-name=org.freedesktop.IBus",
+        "--talk-name=org.freedestkop.portal.*",
+        "--filesystem=home"
+    ],
+    "command": "logarithmplotter",
+    "rename-desktop-file": "logarithmplotter.desktop",
+    "rename-icon": "logplotter",
+    "rename-appdata-file": "eu.ad5001.LogarithmPlotter.metainfo.xml",
+    "cleanup": [ ],
+    "post-install": [
+        "install -D mimetypes /app/share/mime/packages/x-logarithm-plot.xml"
+    ],
+    "modules": [
+        "python3-pyside2.json",
+        {
+            "name": "LogarithmPlotter",
+            "buildsystem": "simple",
+            "config-opts": ["no-debuginfo-compression"],
```

Comment:
What is it needed for?

Reply:
Oh this is a residue from when I tried another method to compile PySide2. I'll remove it.

Reply:
Fixed in latest commit.